pr2eus/robot-interface.l

```diff
@@ -1241,20 +1241,23 @@ Return value is a list of interpolatingp for all controllers, so (null (some #'i
           base-frame-id
           odom-topic move-base-trajectory-joint-names
           go-pos-unsafe-goal-msg
-          current-goal-coords)) ;; for simulation-callback
+          current-goal-coords
+          holonomic)) ;; for simulation-callback
```

Comment:
Comment `;; for simulation-callback` is for `current-goal-coords`, I think.
So please keep them in the same line.

Reply:
I updated